app/controllers/redirect_to_accept_terms_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RedirectToAcceptTermsController < ApplicationController
```

Comment:
I would probably just call this AcceptTermsController, to be in line with the accept_terms route we have. Redirecting is just an assumed part of a controller's responsibility.